packages/stream-model-instance-handler/src/model-instance-document-handler.ts

```diff
@@ -204,6 +204,10 @@ export class ModelInstanceDocumentHandler implements StreamHandler<ModelInstance
    * @private
    */
   async _validateContent(context: Context, modelStreamId: StreamID, content: any): Promise<void> {
+    if (isEmpty(content)) {
+      // We leave the possibility to set content to null/{} as a way of 'soft' deletion of an MID
+      return Promise.resolve()
```

Comment:
```suggestion
      return
```

Reply:
@pawartur does it mean that you could do "soft-delete", and then add content? Is it intended?

Reply:
Good question! I assumed that this is what we meant when we were discussing it.

Is there a way that we can soft delete and prevent any following content changes from that point?

Reply:
We certainly could change the commit handler to not allow setting contents if the current contents are null, but I'm not sure that we actually want to do that?